scripts/local/e2e_test.sh

```diff
@@ -16,6 +16,7 @@ BASEDIR=${BASEDIR:-"$HOME/.teleporter-deps"}
 
 cwd=$(pwd)
 # Install the avalanchego and subnet-evm binaries
+rm -rf $BASEDIR/avalanchego
```

Comment:
A recent change is causing the network artifacts to not fully be cleaned on exit, causing errors on subsequent runs. This works around that.